.circleci/config.yml

```diff
@@ -144,6 +133,23 @@ jobs:
             --repo-url=${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
             --monorepo-tags
 
+  release-please-next:
+    <<: *container_config_node
+    steps:
+      - *attach_workspace
+      - run:
+          name: Update release-please release PR
+          command: npx release-please@15 release-pr --token=${RELEASE_PLEASE_GITHUB_TOKEN}
+            --repo-url=${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
+
+      - run:
+          name: Release any unreleased PR merges
+          command: npx release-please@15 github-release
+            --token=${RELEASE_PLEASE_GITHUB_TOKEN}
+            --repo-url=${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
+            --monorepo-tags
+            --prerelease
```

Comment:
**todo**: think we need to set the [target branch](https://github.com/googleapis/release-please/blob/2e28edfbe237f042e679621f89669f969f0c6a12/docs/cli.md#L19) to `next` here

```suggestion
            --repo-url=${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
            --target-branch next

      - run:
          name: Release any unreleased PR merges
          command: npx release-please@15 github-release
            --token=${RELEASE_PLEASE_GITHUB_TOKEN}
            --repo-url=${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
            --monorepo-tags
            --prerelease
            --target-branch next
```